compensated-spec/features/steps.rb

```diff
@@ -1,13 +1,26 @@
+ParameterType(
+  name: 'spec_helper',
+  regexp: /`.*`/,
+  transformer: ->(s) { s.gsub("`", '') },
+  use_for_snippets: true
+)
+
+
 Given("we have included the `Compensated::Spec::Helpers` mixin in my testing framework") do
   include Compensated::Spec::Helpers
 end
 
-When("we generate the {string} fixture") do |fixture_name|
-  @fixture_output = compensated_event_body(fixture_name)
+When("I call {spec_helper} for a {string} fixture with the following overrides:") do |spec_helper, fixture_name, table|
```

Comment:
Do you intended to use `spec_helper` soon?

Reply:
Yes, as we add feature tests for the other helpers defined in `lib/compensated/spec` I expect we'll want to use a case statement to indicate which method we want to call similar to the `When("I call {spec_helper} for a {string} fixture")` step below.